services/src/main/java/org/apache/unomi/services/impl/events/EventServiceImpl.java

```diff
@@ -139,7 +141,8 @@ public boolean isEventAllowed(Event event, String thirdPartyId) {
     }
 
     public boolean isEventValid(Event event) {
-        return schemaRegistry.isValid(event, "https://unomi.apache.org/schemas/json/events/" + event.getEventType() + "/1-0-0");
+
```

Comment:
Does this method still make sense to keep if all validation was done at the endpoint level ?

Reply:
indeed, removed as suggested